src/security/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Security
+description: >
+  An overview of the Flutter's team philosophy and processes for security.
```

Comment:
Same comment. Is the arrow needed here?

Reply:
In this case I used it to keep the line <= 80 characters.